e2e/expo/src/expo.test.ts

```diff
@@ -22,6 +23,16 @@ describe('expo', () => {
 
   beforeAll(() => {
     proj = newProject();
+    // we create empty preset above which skips creation of `production` named input
+    updateJson('nx.json', (nxJson) => {
+      nxJson.namedInputs = {
+        default: ['{projectRoot}/**/*', 'sharedGlobals'],
+        production: ['default'],
+        sharedGlobals: [],
+      };
+      nxJson.targetDefaults.build.inputs = ['production', '^production'];
+      return nxJson;
+    });
```

Comment:
Hm this seems wrong...

So if I create an `empty` workspace, then create an Angular app, the inputs aren't setup the same way as if I just create an Angular workspace?

Reply:
This is the block where it's set https://github.com/nrwl/nx/blob/master/packages/workspace/src/generators/new/generate-workspace-files.ts#L116-L127

The `angular` has it only as a part of `ng-add` which is external command. All the `init` methods called by generators have something like this https://github.com/nrwl/nx/blob/master/packages/jest/src/generators/init/init.ts#L118-L119

Reply:
if production named input wasn't created during the CNW then none of the generators would add it.